src/components/StatusBar/index.tsx

```diff
@@ -33,7 +40,22 @@ const handleCopyLinkButtonClick = () => {
  */
 const StatusBar = () => {
     const {uiState, numEvents} = useContext(StateContext);
-    const {logEventNum} = useContext(UrlContext);
+    const {isPrettified, logEventNum} = useContext(UrlContext);
+
+    const handleStatusButtonClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+        const {actionName} = event.currentTarget.dataset;
```

Comment:
Let's name it `ev` as a convention in this codebase. Excellent use of the `data-` attribute by the way!

```suggestion
    const handleStatusButtonClick = (ev: React.MouseEvent<HTMLButtonElement>) => {
        const {actionName} = ev.currentTarget.dataset;
```

Reply:
Fixed! Also fixed the `MenuBarIconButton`, it also names it as `event`.